### PR TITLE
Add some tests to increase coverage

### PR DESCRIPTION
**Pull request by angrykoala**
This PR does not change the behavior, but adds some tests to cover some edge cases

### CODE REVIEW

File: src/references/NodeRef.test.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * limitations under the License.
+ */
+
+import Cypher from "..";
+import { TestClause } from "../utils/TestClause";
+
+describe("NodeRef", () => {
+    test("Create node", () => {
+        const rel1 = new Cypher.Node();
+        const rel2 = new Cypher.Node();
```
[the remark this answers is not in the document]
Comment (angrykoala) [reply src/references/NodeRef.test.ts]: Nope, that was a typo :sweat_smile:
